examples/ecs-firelens/container-definitions-fargate.template.json

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "name": "example",
+    "image": "<account-id>.dkr.ecr.<region>.amazonaws.com/<image-name>:latest",
+    "environment": [{
+      "name": "AWS_EMF_AGENT_ENDPOINT",
+      "value": "tcp://127.0.0.1:25888"
```

Comment:
We have set this environment variable so that our application container can send the EMF metrics to this endpoint.
Related documentation - https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/cloudwatchlogs-emf#awsvpc-or-host-mode

Reply:
I think that example assuming we don't use this library - ie how to send EMF raw logs directly through firelens. Since our library already define a TCP client to send to this port, I don't see a reason we need that. Will it work after you remove this?

Reply:
The github example is for Firelens only. However, we have defined the default TCP endpoint to be "127.0.0.1:25888" which will ensure that this will work without specifying the environment variable value in the task definition. Related link - https://github.com/awslabs/aws-embedded-metrics-java/blob/master/src/main/java/software/amazon/cloudwatchlogs/emf/sinks/Endpoint.java#L31